src/plugins/intel_gpu/tests/fusions/convolution_fusion_test.cpp

```diff
@@ -4378,6 +4378,74 @@ INSTANTIATE_TEST_SUITE_P(fusings_gpu, conv_before_permute_optimizing, ::testing:
     convolution_test_params{ CASE_CONV_FP16_PERMUTE_2, 3, 2, 4 },
 }));
 
+class EltwiseSumWithConstantFullTensorFusingTestOneDNN : public BaseFusingTest<convolution_eltw_sum_test_params> {
+public:
+    void execute(convolution_eltw_sum_test_params& p) {
+        if (!engine.get_device_info().supports_immad)
+            return;
+        auto input_prim = get_mem(get_weights_layout(p));
+
+        network network_not_fused(this->engine, this->topology_non_fused, cfg_not_fused);
+        network network_fused(this->engine, this->topology_fused, cfg_fused);
+        network_fused.set_input_data("input", input_prim);
+        network_not_fused.set_input_data("input", input_prim);
+
+        for (int i = 0; i < 10; i++) {
+            network_not_fused.execute();
+            network_fused.execute();
+        }
+
+        compare(network_not_fused, network_fused, p);
+
+        auto pi_fused = network_fused.get_primitives_info();
+        auto info_fused = std::find_if(pi_fused.begin(), pi_fused.end(), [](primitive_info& p) -> bool {
+            if (p.original_id == "conv_prim")
+                return true;
+            return false;
+        });
+
+        if (info_fused != pi_fused.end()) {
+            std::cout << "kernel: " << info_fused->kernel_id << std::endl;
+            EXPECT_TRUE(info_fused->kernel_id.find("jit:ir") != std::string::npos);
+        }
+    }
 
+    layout get_input_layout(convolution_eltw_sum_test_params& p) {
+        auto pad = p.pad;
+        std::vector<int> pad_ = { 0, 0, static_cast<int>(pad[0]), static_cast<int>(pad[1]) };
+        return layout{ p.data_type, p.input_format, p.in_shape, padding{ pad_ } };
+    }
+
+    layout get_per_channel_layout(convolution_eltw_sum_test_params& p) {
+        return layout{ p.default_type, p.default_format, tensor{ 1, p.out_shape.feature[0], 1, 1 } };
+    }
+};
+
+// When dependency of eltwise is full tensor constant, use binary add instead of sum as post-op.
+class onednn_replace_full_tensor_sum_to_binary_add : public EltwiseSumWithConstantFullTensorFusingTestOneDNN {};
+TEST_P(onednn_replace_full_tensor_sum_to_binary_add, basic) {
+    auto p = GetParam();
+    if (engine.get_device_info().supports_immad)
+        p.expected_fused_primitives = p.expected_fused_primitives_onednn;
+
+    create_topologies(
+        data("src0", get_mem(get_input_layout(p))),
+        input_layout("input", get_weights_layout(p)),  // Input is weights.
+        data("eltwise_data", get_mem(layout{ p.eltw_type, p.eltw_format, p.out_shape })),
```

Comment:
as mentioned above, what if we have additional layer between this `eltwise_data` and `sum`? For example, eltwise operation... In that case, is it recognized as constant well?

Reply:
At this line, network::execute() function is called multiple times in execute function of test class. Please see line 4393.